packages/fxa-auth-server/package.json

```diff
@@ -29,7 +29,10 @@
     "paypal-processor": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/paypal-processor.ts",
     "subscription-reminders": "CONFIG_FILES='config/secrets.json' ts-node ./scripts/subscription-reminders.ts",
     "write-templates": "ts-node lib/senders/emails/render-to-disk.ts",
-    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts"
+    "maildev-preview": "ts-node lib/senders/emails/preview-in-maidev.ts",
```

Comment:
Ohh, unrelated to your PR but could be done in a quick follow up - we're missing the "l" in `preview-in-maildev.ts` filename.